### PR TITLE
ft/contact-page

### PR DESCRIPTION
**Pull request by IGIRANEZAJosue**
**Implementation of the contact page**

### CODE REVIEW

**Comment by IGIRANEZAJosue** [comment]
[Imprint page - Desktop](https://trello.com/c/DKvH5Jl5/31-imprint-page-desktop)

**Comment by musi922** [comment]
There is no image in the navbar maybe you can fix that

**Comment by IGIRANEZAJosue** [comment]
[Imprint page - Responsiveness](https://trello.com/c/63yw2Mw4/32-imprint-page-responsiveness)

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="stylesheet" href="/src/output.css" />
+      <title>Contact</title>
+   </head>
+   <body class="font-outfit text-webGreen">
+      <!-- Navbar -->
```
Comment (uwumukiza123) [inline contact.html:10]: Remember to remove comments

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+      <header>
+         <nav
+            class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+         >
+            <div class="flex flex-col md:text-2xl lg:text-2xl">
```
Comment (uwumukiza123) [inline contact.html:15]: I think, this md:text-2xl will also work on the larger screen, you don't need to add another one

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="stylesheet" href="/src/output.css" />
+      <title>Contact</title>
+   </head>
+   <body class="font-outfit text-webGreen">
+      <!-- Navbar -->
+      <header>
+         <nav
+            class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
```
Comment (uwumukiza123) [inline contact.html:13]: Also here this lg:p-6 will also work on the xl screens, you don't need to add

File: contact.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+         >
+            <div class="flex flex-col md:text-2xl lg:text-2xl">
+               <p>DE</p>
+               <p>EN</p>
+            </div>
+            <div class="lg:absolute lg:pl-[40%]">
+               <h1 class="font-semibold text-2xl md:text-5xl">Baumeister</h1>
+            </div>
+            <div
+               class="absolute lg:flex flex-col lg:items-end space-y-10 md:space-y-24 lg:space-y-0 hidden md:ml-[20%] lg:ml-0 pt-[120%] md:pt-[70%] lg:pt-14 pl-32 lg:pl-[87%] font-bold md:font-bold lg:font-normal text-3xl md:text-5xl lg:text-2xl"
```
Comment (uwumukiza123) [inline contact.html:23]: Same as here, font-bold will also work on the md(medium) screens you can remove this md:font-bold

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+            <span class="flex lg:hidden w-8 h-6 cursor-pointer" id="showmenu">
+               <img src="/Images/greenOpenMenu.svg" alt="Menu" />
+            </span>
+            <span class="hidden w-8 h-6 cursor-pointer" id="hidemenu">
+               <img src="/Images/greenCloseMenu.svg" alt="Menu" />
+            </span>
+         </nav>
+      </header>
+
+      <!-- container -->
```
Comment (uwumukiza123) [inline contact.html:41]: Remove comments

File: contact.html (hunk 4 above)
Comment (uwumukiza123) [inline contact.html:23]: Same as here, font-bold will also work on the md(medium) screens you can remove this md:font-bold

File: contact.html [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+               accuracy, completeness and up to-dateness of the information
+               provided. The same applies to third-party websites to which
+               www.agentur-baumeister.com refers by means of hyperlinks. Katrein
+               Baumeister, Agentur Baumeister accepts no liability for the
+               content of these websites.
+            </p>
+         </div>
+      </div>
+
+      <!-- Red image -->
```
Comment (uwumukiza123) [inline contact.html:84]: Same as here, you can delete the comments

File: contact.html [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+         <div class="md:flex justify-center items-center hidden px-8 py-24">
+            <img
+               class="w-auto md:w-3/4 lg:w-1/2"
+               src="/Images/redShape.svg"
+               alt="redShape"
+            />
+         </div>
+      </section>
+
+      <!-- Footer -->
```
Comment (uwumukiza123) [inline contact.html:95]: Same as here

File: src/sections/Navbar.html [hunk 8]
```diff
@@ -1,27 +1,27 @@
...
-  <div
-    class="absolute font-bold text-3xl space-y-10 md:space-y-24 lg:space-y-0 lg:flex flex-col lg:items-end hidden md:ml-[20%] lg:ml-0 pt-[120%] md:pt-[70%] md:text-5xl md:font-bold lg:pt-14 pl-32 lg:pl-[87%] lg:font-normal lg:text-2xl text-green-700 lg:text-black"
-    id="menu"
-  >
-    <p>About</p>
-    <p>Services</p>
-    <p>References</p>
-    <p>Contact</p>
-  </div>
+   <div class="flex flex-col md:text-2xl lg:text-2xl">
```
Comment (uwumukiza123) [inline src/sections/Navbar.html:4]: This md:text-2xl will also work on larger screens, you can remove this lg:text-2xl

**Review by uwumukiza123 — APPROVED** [review]

File: src/sections/Navbar.html [hunk 9]
```diff
@@ -1,27 +1,27 @@
 <nav
-  class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit lg:mb-12 sticky top-0 z-10"
+   class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
```
Comment (uwumukiza123) [inline src/sections/Navbar.html:2]: Same as here, this lg:p-6 will also work on the xl screens

File: src/sections/Navbar.html [hunk 10]
```diff
@@ -1,27 +1,27 @@
...
-  </div>
+   <div class="flex flex-col md:text-2xl lg:text-2xl">
+      <p>DE</p>
+      <p>EN</p>
+   </div>
+   <div class="lg:absolute lg:pl-[40%]">
+      <h1 class="font-semibold text-2xl md:text-5xl">Baumeister</h1>
+   </div>
+   <div
+      class="absolute lg:flex flex-col lg:items-end space-y-10 md:space-y-24 lg:space-y-0 hidden md:ml-[20%] lg:ml-0 pt-[120%] md:pt-[70%] lg:pt-14 pl-32 lg:pl-[87%] font-bold md:font-bold lg:font-normal text-3xl text-green-700 md:text-5xl lg:text-2xl lg:text-black"
```
Comment (uwumukiza123) [inline src/sections/Navbar.html:12]: Here as well font-bold will also work on medium screens

**Review by uwumukiza123 — CHANGES_REQUESTED** [review]
You did a great job, but there are some minor changes

**Review by uwumukiza123 — APPROVED** [review]

**Review by Algor-Fernand — APPROVED** [review]
This looks fine to me. Good job 🎉🎉

**Pull request merged**